packages/simplebar/src/scrollbar-width.js

```diff
@@ -12,15 +13,16 @@ if (canUseDOM) {
   });
 }
 
-export default function scrollbarWidth() {
+export default function scrollbarWidth(element) {
+  const elementDocument = getElementDocument(element);
```

Comment:
I don't think this is needed in this case. This helper is to calculate the native scrollbar width which is going to be the same in any document of the environment so I believe you don't need to use the helper in this case.